kie-server-parent/kie-server-tests/kie-server-integ-tests-all/pom.xml

```diff
@@ -138,6 +138,7 @@
               <org.kie.prometheus.server.ext.disabled>${org.kie.prometheus.server.ext.disabled}</org.kie.prometheus.server.ext.disabled>
               <org.kie.scenariosimulation.server.ext.disabled>${org.kie.scenariosimulation.server.ext.disabled}</org.kie.scenariosimulation.server.ext.disabled>
               <kie.server.testing.kjars.build.settings.xml>${kie.server.testing.kjars.build.settings.xml}</kie.server.testing.kjars.build.settings.xml>
+              <org.kie.server.bypass.auth.user>true</org.kie.server.bypass.auth.user>
```

Comment:
This property is not necessary here, you can remove it

Reply:
actually, it is needed here for the client because of the way kie-server client is implemented. See below code in the client:

- https://github.com/kiegroup/droolsjbpm-integration/blob/main/kie-server-parent/kie-server-remote/kie-server-client/src/main/java/org/kie/server/client/impl/AbstractKieServicesClientImpl.java#L697-L702

If the property is not set, then we will be sending an empty user id which is incorrect. 

Having said that, I think this is a wrong setup in the kie-server client since this `BYPASS_USER` property is only required at server level, but that's a different topic than what this PR is trying to fix

wdyt?